example/way_to_make_startup.py

```
@@ -0,0 +1,27 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)